platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## master
+
+### Styles and rendering
+
+* Implemented asymmetric center of perspective: the viewport center is in the center of the area specified by [MGLMapView contentInset](https://docs.mapbox.com/ios/api/maps/5.0.0/Classes/MGLMapView.html#/c:objc(cs)MGLMapView(py)contentInset) and vanishing point is above it. [#14664](https://github.com/mapbox/mapbox-gl-native/pull/14664)
```

Comment:
may I suggest:
> * Fixed an issue that caused the focal point to be always based on the view's horizontal center when setting `MGLMapView.contentInset`. [#14664](https://github.com/mapbox/mapbox-gl-native/pull/14664)

Reply:
Thanks. Combined to: Implemented asymmetric center of perspective: fixed an issue that caused the focal point to be always based on the view's horizontal center when setting ...